CIP-0100/README.md

```diff
@@ -0,0 +1,148 @@
+of the specific examples mentioned earlier.
+This CIP will collect and list all widely used validators.
+
+#### List of supercharged validator tags, with specification
+
+1. `#000001`: "Always burning" Minting Validators, that always validates if all amounts of minted assets of the governed policy id are negative (i.e. being burned)
+
+#### Representation of Native Scripts as PlutusData
+
+The standard representation of Native Scripts as PlutusData objects can be derived from the following specification (in PyCardano `PlutusData` notation):
+
+```python
+@dataclass()
+class RequireSignature(PlutusData):
+    CONSTR_ID = 0
+    vkeyhash: bytes  # this is either a PubKeyHash or a VerificationKeyHash
+
+
+@dataclass()
+class RequireAllOf(PlutusData):
+    CONSTR_ID = 1
+    scripts: List[Datum]  # "Script"
+
+
+@dataclass()
+class RequireAnyOf(PlutusData):
+    CONSTR_ID = 2
+    scripts: List[Datum]  # "Script"
+
+
+@dataclass()
+class RequireMOf(PlutusData):
+    CONSTR_ID = 3
+    num: int
+    scripts: List[Datum]  # "Script"
+
+
+@dataclass()
+class RequireBefore(PlutusData):
+    CONSTR_ID = 4
+    unixtimestamp: int
+
+
+@dataclass()
+class RequireAfter(PlutusData):
+    CONSTR_ID = 5
+    unixtimestamp: int
+
+
+Script = Union[
+    RequireSignature,
+    RequireMOf,
+    RequireAnyOf,
+    RequireAllOf,
+    RequireAfter,
+    RequireBefore,
+]
+```
+
+## Rationale
+
+The main motivation for this proposal is the problem of ADA being locked by the minUTxO requirements of the ledger together with native tokens.
+Native tokens at a UTxO increase it's size and hence increase the mininmum attached lovelace of the output.
+This is generally as desireable as it is intended since this bounds the size of the UTxO set by the given amount of lovelace.
+
+The ADA could be unlocked by removing the tokens from the ledger altogether by burning them.
+One option to achieve this is to make all native tokens burnable by their owners.
+This leads to a number of necessary changes in the ledger and also violates security guarantees for early adopters of native tokens - generally this proposal suggests to go a different route.
+
+
+The proposed solution will not force all tokens to be burnable, allowing creators to opt-in for burnable tokens when desired. 
+We suggest that token creators are given the freedom to use the intuitive native token interface known so far and
+simple tooling to supercharge it with the available Plutus completeness.
+In order to make sure that most users can benefit from this, the proposal suggests that the future *default* for token creation is choosing
+a specific type of supercharged native scripts.
+
+By replacing native scripts with smart contracts for minting and burning tokens, users will have a simple and standardized way to manage their assets and recover locked ADA.
```

Comment:
You can already just use a Plutus script for this, so I'm confused about what this adds.